### PR TITLE
Account and Campaign API Fixes.

### PR DESCRIPTION
**Pull request by rlr**
* add store_missing=False to name in case of PUT for campaigns
* make account_id required when requesting campaign list

r? @ncloudioj

### CODE REVIEW

File: splice/web/api/campaign.py [hunk 1]
```diff
@@ -89,7 +89,11 @@ def get(self):
 
         Takes an optional account_id as a query string argument.
         """
-        campaigns = get_campaigns(request.args.get('account_id'))
+        parser = reqparse.RequestParser()
```
Comment (ncloudioj) [inline splice/web/api/campaign.py]: better create one single RP instance in `__init__` and reuse it. i know premature optimization is the root of evil :)

Comment (rlr) [inline splice/web/api/campaign.py]: but the other method uses a different parser (for POST). That's why it's local here.

Comment (ncloudioj) [inline splice/web/api/campaign.py]: I just created a self.reqparse_get for this. Totally up to you, not an issue at all.

**Comment by ncloudioj** [comment]
r+ @rlr

File: splice/web/api/campaign.py (hunk 1 above)
Comment (rlr) [inline splice/web/api/campaign.py]: ah I see. I'll do that for consistency.

**Comment by rlr** [comment]
Pull request 100!!!!!!! :tada: 
fff0a262ca4243a3218a024f6cc4259b76d3c0c3

**Pull request closed without merging**